aiohttp/resolver.py

```diff
@@ -26,6 +27,15 @@ def __init__(self, loop=None):
         self._loop = loop
 
     async def resolve(self, host, port=0, family=socket.AF_INET):
+        if family == 0:
```

Comment:
What about to use an auto descriptive constant such as `ALL_FAMILIES`? This would help the user.

Reply:
Done. `socket.AF_UNSPEC` has `0` value.